libraries/chain/include/eosio/chain/producer_schedule.hpp

```diff
@@ -46,24 +46,28 @@ namespace eosio { namespace chain {
       };
    }
 
+   struct block_signing_authority_v0;
+
    struct shared_block_signing_authority_v0 {
-      shared_block_signing_authority_v0() = delete;
+      shared_block_signing_authority_v0() = default;
+      shared_block_signing_authority_v0( const block_signing_authority_v0& );
```

Comment:
`explicit`

Reply:
Thanks, made the constructor at line 52 explicit. Other one is very specific.

Reply:
`explicit` on a default constructor has no affect, right?

Reply:
You are right, what was I thinking! Will remove the explicit.